### DISCUSSION TITLE
Fans running at almost max rpm on low percentage duty cycle

### DISCUSSION THREAD

**Discussion by Chipthemunk** [tjko/fanpico#54] [Q&A] [answered]
Hi I am having issues trying to get the fans (ACFAN00125A) to run at appropriate rpm when told they should use sensor1 with temp map.

Fan operating rpm is from 200 to 1700

Running: TJKO Industries,FANPICO-0804D,e6616408437bad38,1.5.0

I have not connected the display yet.

output from `read?` is:

mbfan1,"mbfan1",1501,50.03,100.0
mbfan2,"mbfan2",1531,51.02,100.0
mbfan3,"mbfan3",1524,50.79,100.0
mbfan4,"mbfan4",1652,55.08,100.0
fan1,"fan1",1501,50.03,12.6
fan2,"fan2",1531,51.02,12.6
fan3,"fan3",1524,50.79,12.6
fan4,"fan4",1652,55.08,12.6
fan5,"fan5",1496,49.85,12.6
fan6,"fan6",1652,55.08,12.6
fan7,"fan7",0,0.00,100.0
fan8,"fan8",0,0.00,100.0
sensor1,"Fluid Temp",26.3,12.6
sensor2,"sensor2",-22.6,0.0
sensor3,"pico_temp",38.0,60.1
vsensor1,"vsensor1",0.0,0.0
vsensor2,"vsensor2",0.0,0.0
vsensor3,"vsensor3",0.0,0.0
vsensor4,"vsensor4",0.0,0.0
vsensor5,"vsensor5",0.0,0.0
vsensor6,"vsensor6",0.0,0.0
vsensor7,"vsensor7",0.0,0.0
vsensor8,"vsensor8",0.0,0.0

output from `conf:sensor1:tempmap?` is:

20.000000,0.000000,30.000000,20.000000,35.000000,33.000000,40.000000,50.000000,45.000000,70.000000,50.000000,90.000000,55.000000,100.000000

Config is attached 
[currentconfig.txt](https://github.com/tjko/fanpico/files/12446457/currentconfig.txt)

**Comment by Chipthemunk** [comment, marked as answer]
if you have aux power plugged in but set the main power jumper to "fan" that will cause the symptoms  mentioned above.

**Reply by tjko** [reply]
For fan control to work there must be 12V coming in either via MBFAN1 or AUX connector and the main power jumper set accordingly.
(If both AUX and MBFAN1 are connected, then jumper can be set either way...)

Looks like silkscreen is wtong for main power jumper should have "mbfan1" not "fan1"...
